Dockerfile

```diff
@@ -3,28 +3,23 @@ FROM ubuntu:18.04
 LABEL mantainer="Read the Docs <support@readthedocs.com>"
 LABEL version="5.0.0rc1"
 
-ENV DEBIAN_FRONTEND noninteractive
```

Comment:
Yes, that's the reason why it's done via `export ...`.

Reply:
I mean, running 

    docker run -it readthedocs/build:latest /bin/bash

I suppose that we still want to run with the `noninteractive`, don't we?

Reply:
Running it with `docker run -i` implies to be interactive, setting `DEBIAN_FRONTEND=noninteracive` is contradicting. :wink: 


See the official answer https://github.com/moby/moby/issues/4032#issuecomment-34597177

Reply:
Yeah, I know, but I want to have the same behavior when running `apt install X` from `docker run -i` than when that command is ran from Read the Docs build process internally (a `os.system` call at `setup.py` for example). Otherwise, it's hard to be sure that what I'm testing locally will work in production.